requirements-test.txt

```diff
@@ -1,5 +1,5 @@
-Click==7.0
+click==7.0
 ruamel.yaml==0.15.89
 flake8==3.7.7
-pytest>=3.5.0
-pytest-subtests==0.3.0
+pytest>=4.4
```

Comment:
Just a question: Is there a specific reason that you choose version 4.4?

Reply:
Actually while using pytest-subtest the minimum requirement is [Pytest Version 4.4](https://github.com/pytest-dev/pytest-subtests)

Reply:
Got it, thank you.